src/Core/AdminConsole/OrganizationFeatures/Policies/Implementations/SavePolicyCommand.cs

```diff
@@ -75,6 +80,8 @@ public async Task<Policy> SaveAsync(PolicyUpdate policyUpdate)
         await _policyRepository.UpsertAsync(policy);
         await _eventService.LogPolicyEventAsync(policy, EventType.Policy_Updated);
 
+        await PushPolicyUpdateToClients(policy.OrganizationId, policy);
```

Comment:
We need to add this functionality to the [VNextSavePolicyCommand](https://github.com/bitwarden/server/blob/c3847db9f898fccdecf32a21e2aa729e94b6092b/src/Core/AdminConsole/OrganizationFeatures/Policies/Implementations/VNextSavePolicyCommand.cs) as well